kodilla-rps/src/main/java/com/kodilla/rps/GameService.java

```diff
@@ -0,0 +1,178 @@
+package com.kodilla.rps;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class GameService {
+    Random random = new Random();
+    Scanner scanner = new Scanner(System.in);
+
+    boolean gameOver = false;
+
+    String playerNumber;
+    int cpuRandomNumber;
+
+    int rounds = 0;
+    int countOfRounds = 0;
+    int cpuWinCounter = 0;
+    int playerWinCounter = 0;
+
+    public void playGame() {
+
+        System.out.println();
+        System.out.println("Welcome to the game paper, rock, scissors!\nHow many rounds do you want to play? Enter from 1 to 5:");
+
+        rounds = getNumbersOfRound();
+        System.out.println("We're starting the game!\n");
+        System.out.println("Select the number:\n paper = 1 \n stone = 2 \n scissors = 3 \n exit = x \n new game = n");
+
+        while (!gameOver && rounds > 0) {
+            System.out.println();
+            System.out.println("Round no. " + ++countOfRounds);
+
+            getCorrectNumber();
+            getUserMove(playerNumber);
+
+            if (!gameOver) {
+                System.out.println();
+                cpuRandomNumber = random.nextInt(3) + 1;
+                draw(cpuRandomNumber);
+                System.out.println();
+
+                String resultOfTheRound = checkWinner(playerNumber, cpuRandomNumber);
+                System.out.println(resultOfTheRound);
+
+                rounds--;
+                String gameResult = printCurrentResult(cpuRandomNumber, playerNumber, rounds);
+                System.out.println(gameResult);
+            }
+        }
+        gameOver = true;
```

Comment:
po co to ustawianie zmiennej juz poza petla?

Reply:
bez tego po użyciu "n" na koniec gry wyświetla się: 
Same :)
Congratulations you won! player 1 : cpu 0
End of the game!  
po czym komputer losuje dalej, wyświetla rysunek i napis np.: 
The scissors cut paper!
player 2 : cpu 0
End of the game!